### PR TITLE
Removes Assistants, Assistants are now only available when no other jobs are

### PR DESCRIPTION
**Pull request by KazeEspada**
Stems the tide once and for all!
Assistants are not a job. They can never be made fun by code, and often this leads to "greytiding". For every one gimmick assistant, there are probably about 4 more assistants who are just going to be shits for the sake of it.

Besides, everyone complaining about wasting job slots doing nothing:
Atmos Techs
Librarian
Lawyers
Cargo Techs
Chaplain
The third scientist
The chef(especially the second chef)
Clown and Mime
The second and third botanist slot.
Engineering after round start(set up the engine and then you are just a glorified assistant).
The third doctor.
Miners is a very chill job.

Also, new players learn NOTHING from being an assistant that they couldn't learn something more useful in another job.
:cl: KazeEspada
remove:Nanotrasen has made some budget cuts and is trying to improve efficiency of their stations, they have removed the ability to sign up as assistant unless all jobs are full.
/:cl:

### CODE REVIEW

**Comment by Cheridan** [comment]
wow very good code cleanings :+1: :+1:

**Comment by Giacom** [comment]
Is ```userandomjob``` not used anymore?

**Comment by lzimann** [comment]
won't this remove the random job option?

**Comment by Giacom** [comment]
At a glance of this PR, it looks like it would. Would be great if the PR title and description was changed.

**Comment by RemieRichards** [comment]
@Giacom Yes, he's going to do it.

**Comment by RemieRichards** [comment]
That should re-announce it properly.

**Comment by Thunder12345** [comment]
:-1:
New players and people who want to play a silly gimmick: Taking up job slots since 2016

**Comment by RemieRichards** [comment]
Oh and my stance is :-1: because assistant is my chill "no worries" job.

**Comment by Thunder12345** [comment]
Yeah, you can't stop assistants assistanting, they'll just take up job slots to do it like this.

**Comment by TheRealGLH** [comment]
If I can't be a useless piece of shit as an assistant, I will just waste a job slot while being just as useless.

**Comment by CosmicScientist** [comment]
:-1: Off with his head!

**Comment by KazeEspada** [comment]
@TheRealGLH @Thunder12345 @RemieRichards Yeah, you could all "waste" a librarian or atmos tech slot being just as useless.

**Comment by lzimann** [comment]
big no for this :-1:

**Comment by Qbopper** [comment]
Trying to stop greytide by removing assistant doesn't make much sense to me

If someone is going to be a shit they'll take up a job slot and be shit

This isn't an appropriate reaction, just because there's a lot of shitty players who fuck about as assistants doesn't mean the role should be removed - what if I want to sit back and play some SS13 but just chill out and walk around the station? If I have an actual job I shouldn't be doing that. Sure, you say the ratio of gimmick assistants to shitters is poor, but let's be honest, the number of shitters will stay high no matter what, and I'd rather have them stay assistants instead of taking up actual jobs

Big thumbs down from me

**Comment by TheRealGLH** [comment]
If you think people are being shitters for the sake of it, just pester the admins to fucking ban them.

**Comment by GunHog** [comment]
Is this a serious PR? While I personally **HATE** most assistants with a passion (the ones that break into places, disrupt work, and steal things), I do not believe at all that this will solve any problems. 

You cannot remove the terrible, awful assistant players in this manner. They will simply become terrible, awful engineers and cargo techs. The job-ban rate will increase, there is no doubt here, but not to match the increase in awful play. At least most assistant players have the decency to make it obvious that they are filthy, thieving trash. A cold-turkey removal like this will simply spread them out to other, useful job slots - resulting in a net competence reduction.

Job slots **are** precious - you will understand this when you play as CMO with terrible or no MDs, as an RD with no one bothering to do the research or tending the borgs, and especially as a HoS with only two officers both of which end up to be gibbering morons. Sure, you can try to replace them through the HoP, but there is a second can of worms involved in this process.

**TL;DR Version:** You are not getting RID of the trash, you are just spreading it around the station, and dressing it up.

**Comment by KorPhaeron** [comment]
Man if only we had a security force on station that could arrest annoying players

**Comment by Firecage** [comment]
:-1: Remove KazeEspada

**Comment by KorPhaeron** [comment]
Very lazy "fix" to a symptom of an actual problem.

**Comment by PKPenguin321** [comment]
We already did this before (actually iirc it was paprika who did it) not too long ago. It got reverted because it was shit

**Comment by GunHog** [comment]
I thought Paprika changed all of the assistant uniforms to several different colors. That one was reverted.

**Comment by PKPenguin321** [comment]
He did that too. He did both then I'm pretty sure they were both made config options.

**Comment by GunHog** [comment]
I found options for an assistant cap, assistant rainbow jumpsuits, maint access (sec, assistant, everyone), and antag protection for assistants.

Activating that last option would be pandemonium.

**Comment by bgobandit** [comment]
We tried this. It didn't work. It just led to people using non-assistant jobs to tide.

**Comment by Jazaen** [comment]
> We already did this before (actually iirc it was paprika who did it) not too long ago. It got reverted because it was shit

Apparently TG has it's own version of the Godwins Law. Just replace Austrian painter with pap.
Not that this makes this any better, really. Shit people will be shit, and forcing them into roles where they steal job slots and get more access to tide is not a best way of dealing with them. It's easier to tell sec to "remove this assistant from the medbay now" than "remove this shitty doctor", since assistants are not meant to go there in the first place.

**Comment by ShadowDeath6** [comment]
>remove useless meme job where you play as a shitler

Finally removing clown?

**Comment by RemieRichards** [comment]
Fatality!

**Comment by GunHog** [comment]
Welp! There is always the config route.

**Comment by KorPhaeron** [comment]
Fix the actual problem by making civilian life on the station engaging enough that people don't have to break things to create their own fun.

**Comment by Jazaen** [comment]
Well, that is problematic, since each time PvE comes up people nerf the hell out of it in about a week or two. Or overuse the rewards to the point it has to be disabled.

**Comment by Dawgas** [comment]
I think that's because most of the people who code and administrate are too scared to wait some things out, sometimes - everything new is always jumped on and explored before eventually gathering dust

but assistant is literally the best part of the game, and looking to remove it in this manner shows a lack of understanding about ss13 in general

**Comment by KorPhaeron** [comment]
I realize it's a very difficult problem to resolve, but I think it's a critical one.

**Comment by Jazaen** [comment]
Well, most of the PvE stuff is used just to get new ways of fighting antags or improving survivability (genetics, robotics, good part of R&D, viro), with pretty much only Botany and Xenobio being progression-based departments with small PvP potential. (Well, xenobio, not so much, but it still isn't usually done to fight antags)
